tests/test_all_fields.py

```diff
@@ -93,8 +93,9 @@ def test_run(self):
                 if stream == 'abandoned_checkouts':
                     expected_all_keys.remove('billing_address')
                 elif stream == 'orders':
-                    # No field named 'order_adjustments' present in the 'order' object
+                    # No field named 'order_adjustments', 'total_price_usd' present in the 'order' object
                     #   Documentation: https://shopify.dev/api/admin-rest/2021-10/resources/order#resource_object
-                    expected_all_keys.remove('order_adjustments')
+                    known_missing_fields = ['order_adjustments','total_price_usd']
+                    expected_all_keys= expected_all_keys-set( known_missing_fields)
```

Comment:
https://pep8.org/#other-recommendations

The equals sign should always have spaces around it for when it's in an assignment statement
```suggestion
                    known_missing_fields = ['order_adjustments','total_price_usd']
                    expected_all_keys = expected_all_keys-set( known_missing_fields)
```

Operators like minus should always have spaces around it
```suggestion
                    known_missing_fields = ['order_adjustments','total_price_usd']
                    expected_all_keys= expected_all_keys - set( known_missing_fields)
```

There shouldn't be a space after the open paren of a function call
```suggestion
                    known_missing_fields = ['order_adjustments','total_price_usd']
                    expected_all_keys= expected_all_keys-set(known_missing_fields)
```

A comma and whitespace should separate items in a list. Since it's on one line here, we should use `, `.
```suggestion
                    known_missing_fields = ['order_adjustments', 'total_price_usd']
                    expected_all_keys= expected_all_keys-set( known_missing_fields)
```